### PR TITLE
Add home page and header

### PR DESCRIPTION
**Pull request by faisalrazap**
https://app.asana.com/0/68350206596631/69352653871227

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -8,6 +8,8 @@
 </head>
 <body>
 
+<%= render 'layouts/header' %>
```
Comment (Bilkhaan) [inline app/views/layouts/application.html.erb]: laka indentation may be?

File: app/views/layouts/_messages.html.erb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<div class='messages'>
+  <% flash.each do |key,message| %>
+    <div class = "<%= key %>"><%= message %></div>
```
Comment (Bilkhaan) [inline app/views/layouts/_messages.html.erb]: do we really need double qoutes here?

Comment (faisalrazap) [inline app/views/layouts/_messages.html.erb]: Yeah right, key also returns a string.

**Comment by Bilkhaan** [comment]
looks good

**Pull request merged**